compose/file-watch.md

```diff
@@ -0,0 +1,96 @@
+---
+description: File watch automatically updates running services as you work
+keywords: compose, file watch, experimental 
+title: Automatically update services with file watch
+---
+
+> **Note**
+>
+> The Compose file watch feature is currently [Experimental](../release-lifecycle.md).
+
+Use `watch` to automatically update your running Compose services as you edit and save your code. 
+
+`watch` adheres to the following file path rules:
+
+* All paths are relative to the build context
+* Directories are watched recursively
+* Glob patterns are not supported
+* Rules from `.dockerignore` apply
+  * Use `include` / `exclude` to override
+  * Temporary/backup files for common IDEs (Vim, Emacs, JetBrains, & more) are ignored automatically
+* `.git` directories are ignored automatically
+
+## Configuration
+
+The `watch` attribute defines a list of rules that control automatic service updates based on local file changes.
+
+Each rule requires, a `path` pattern and `action` to take when a modification is detected. There are two possible actions for `watch` and depending on
+the `action`, additional fields might be accepted or required. 
+
+### `action`
+
+#### Sync
+
+If `action` is set to `sync`, Compose makes sure any changes made to files on your host automatically match with the corresponding files within the service container.
+
+Sync is ideal for frameworks that support "Hot Reload" or equivalent functionality.
+
+More generally, they can be used in place of bind mounts for many development use cases.
```

Comment:
Is it worth expanding on this point slightly?

Reply:
New section JUST for comparison to bind mounts added ✨